clients/fidesui/src/icons/GreenCheckCircle.tsx

```diff
@@ -10,7 +10,7 @@ export default createIcon({
   },
   path: (
     <path
-      fill="#38A169"
+      fill="#5A9A68"
```

Comment:
this got missed during recent color updates, just giving it our nicer green color